src/Umbraco.Infrastructure/PropertyEditors/CheckBoxListPropertyEditor.cs

```diff
@@ -24,22 +21,35 @@ public class CheckBoxListPropertyEditor : DataEditor
     {
         private readonly ILocalizedTextService _textService;
         private readonly IIOHelper _ioHelper;
+        private readonly IEditorConfigurationParser _editorConfigurationParser;
+
+        // Scheduled for removal in v12
+        [Obsolete("Please use constructor that takes an IEditorConfigurationParser instead")]
+        public CheckBoxListPropertyEditor(
+            IDataValueEditorFactory dataValueEditorFactory,
+            ILocalizedTextService textService,
+            IIOHelper ioHelper)
+            : base(dataValueEditorFactory)
+        {
+        }
```

Comment:
I'm guessing we should call the non obsoleted constructor and resolve the missing dependencies with the static service provider here?